dart/pubspec.yaml

```diff
@@ -7,18 +7,18 @@ homepage: https://docs.sentry.io/platforms/dart/
 repository: https://github.com/getsentry/sentry-dart
 
 environment:
-  sdk: ">=2.8.0 <3.0.0"
+  sdk: '>=2.12.0-259.8.beta <3.0.0'
```

Comment:
This should be `sdk: '>=2.12.0-0 <3.0.0'`

Reply:
Update: As per https://medium.com/dartlang/preparing-the-dart-and-flutter-ecosystem-for-null-safety-e550ce72c010 this should now be `>=2.12.0–259.9.beta < 3.0.0`

Reply:
```suggestion
  sdk: '>=2.12.0-259.9.beta <3.0.0'
```